realis.go

```diff
@@ -59,6 +59,8 @@ type Realis interface {
 
 	// Admin functions
 	DrainHosts(hosts ...string) (*aurora.Response, *aurora.DrainHostsResult_, error)
+	EndMaintenance(hosts ...string) (*aurora.Response, *aurora.EndMaintenanceResult_, error)
+	MaintenanceStatus(hosts ...string) (*aurora.Response, *aurora.MaintenanceStatusResult_, error)
```

Comment:
a bit weird to have *aurora.Response and *aurora.MaintenanceStatusResult_ in return tuple... Probably it was done to simplify handling on client side -- avoid looking for MaintenanceResult_ from Response object..

Reply:
yup, the idea here is that the response library won't be needed to process the result any longer for newer api. This idea can also be back ported, but for now I'm against changing the signature because it'll cause breakage. I toyed around with the idea of not returning response, but decided against it as we do it right now and the context of the response might be important to some users. The reasons to keep it were stronger, as we can just discard the value by using `_` where the variable name would be on the receiving end.